sky/clouds/service_catalog/data_fetchers/requirements.txt

```diff
@@ -4,3 +4,4 @@ lxml
 pandas
 ray
 requests
+cudo-compute
```

Comment:
We probably don't want to have the cloud specific requirements listed here. We can have it in the `setup.py` for the extra `cudo`. : )